### PR TITLE
docs: edits to samples and ssl_connections topics

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: docs/src/ssl_connections.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+database that's owned by a user called app. (You can change this convention by way of the `initdb`
+configuration in the `bootstrap` section.)
 
 ## Issuing a new certificate
 
 !!! Seealso "About CNPG plugin for kubectl"
-    Please refer to the ["Certificates" section in the "CloudNativePG Plugin"](kubectl-plugin.md#certificates)
-    page for details on how to use the plugin for `kubectl`.
+    See [Certificates in the CloudNativePG plugin](kubectl-plugin.md#certificates)
+    content for details on how to use the plugin for kubectl.
```
Comment (jsilvela) [inline docs/src/ssl_connections.md:22]: this sentence doesn't quite read right

File: docs/src/ssl_connections.md [hunk 2]
```diff
@@ -175,12 +175,10 @@ and [`ssl_max_protocol_version`](https://www.postgresql.org/docs/current/runtime
 to `TLSv1.3`.
 
 !!! Important
-    In PostgreSQL 11 these two GUC doesn't exists hence, in these specific version
-    these values aren't set and will keep using the default ones.
+    In PostgreSQL 11, these two GUCs don't exist. Hence, in these specific versions
+    these values aren't set and continue using the default ones.
```
Comment (jsilvela) [inline docs/src/ssl_connections.md:179]:
```suggestion
    these values aren't set,  and the default ones are used.
```

**Review by jsilvela — CHANGES_REQUESTED** [review]
Nice.
There's a sentence I find weird, and I left a GH suggestion.

File: docs/src/ssl_connections.md (hunk 1 above)
Comment (ebgitelman) [reply docs/src/ssl_connections.md:22]:
```suggestion
    See the [Certificates in the CloudNativePG plugin](kubectl-plugin.md#certificates)
    content for details on how to use the plugin for kubectl.
```

**Comment by ebgitelman** [comment]
Thank you for your thorough review!

**Review by jsilvela — APPROVED** [review]

**Comment by jsilvela** [comment]
thank you!

**Review by gbartolini — APPROVED** [review]

**Pull request merged**